board/safety/safety_toyota.h

```diff
@@ -243,13 +248,27 @@ static int toyota_tx_hook(CANPacket_t *to_send, bool longitudinal_allowed) {
         }
       }
 
-      // no torque if controls is not allowed or mismatch with STEER_REQUEST bit
-      if ((!controls_allowed || !steer_req) && (desired_torque != 0)) {
+      // handle steer_req bit mismatches: we set the bit to 0 at an expected
+      // interval to bypass an EPS fault, violation if we exceed that frequency
+      bool steer_req_mismatch = (desired_torque != 0) && !steer_req;
+      if (!steer_req_mismatch) {
+        toyota_steer_req_matches = MIN(toyota_steer_req_matches + 1U, 255U);
+      } else {
+        // disallow torque cut if not enough recent matching steer_req messages
+        if (toyota_steer_req_matches < (TOYOTA_MAX_STEER_RATE_FRAMES - 1U)) {
```

Comment:
let's also add a time based check to prevent bypassing this by sending at a faster than expected frequency